aiosmtpd/__init__.py

```diff
@@ -1,4 +1,18 @@
 # Copyright 2014-2021 The aiosmtpd Developers
 # SPDX-License-Identifier: Apache-2.0
+import asyncio
+import warnings
 
-__version__ = "1.4.4a0"
+
+__version__ = "1.4.4a1"
+
+
+def _get_or_new_eventloop() -> asyncio.AbstractEventLoop:
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore")
+        try:
+            loop = asyncio.get_event_loop()
+        except (DeprecationWarning, RuntimeError):  # pragma: no cover
```

Comment:
When 3.12 comes out we might have to revisit this and change the caught exception in accordance to the actual exception that will be raised.